table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -211,18 +207,7 @@ public SyncStatus sync(JournalContext context) throws IOException {
     // Synchronization is necessary if accessed concurrently from multiple threads
     SyncStatus.Builder builder = SyncStatus.newBuilder();
 
-    if (!mConfigPath.equals(CatalogProperty.DB_CONFIG_FILE.getDefaultValue())) {
-      if (!Files.exists(Paths.get(mConfigPath))) {
-        throw new FileNotFoundException(mConfigPath);
-      }
-      ObjectMapper mapper = new ObjectMapper();
-      try {
-        mDbConfig = mapper.readValue(new File(mConfigPath), DbConfig.class);
-      } catch (JsonProcessingException e) {
-        LOG.error("Failed to deserialize UDB config file {}, stays unsynced", mConfigPath, e);
-        throw e;
-      }
-    }
+    loadFilterSpec();
```

Comment:
Add some description here

Reply:
do we need to load filter spec everytime on `sync`?

Reply:
> do we need to load filter spec everytime on sync?

The original intention by @maobaolong is to make the config file part of the synchronization job.  https://github.com/Alluxio/alluxio/pull/13455#discussion_r638114342

Reply:
> Add some description here

Added in 2361b59807ed5d1b5cc50ef3e1e70391699c87d8